packages/core/src/configuration.ts

```diff
@@ -146,6 +146,7 @@ const axisChart: AxisChartOptions = Tools.merge({}, chart, {
 	axes,
 	timeScale,
 	grid,
+	scatterDotEnabled: true,
```

Comment:
probably under the `points` flag as `enabled`?

![image](https://user-images.githubusercontent.com/14989804/91452901-2212ba80-e84d-11ea-91f1-5f5119d1ad71.png)

Reply:
if we move `scatterDotEnabled` under the `points `flag as `enabled`, then only scatter chart can access the config option. However, axis charts, such as line chart and area chart, also need to access this config option as well.

Reply:
Don't think this'd be the case. Any chart that uses scatter should extend scatter chart and their options should do the same

Reply:
<img width="733" alt="圖片" src="https://user-images.githubusercontent.com/23396970/91837296-9ea6fe00-ec7e-11ea-9b9f-44ae87b6aaff.png">

Sometimes we don't want to have the scatter dot displayed. That's the reason why I setup this `scatterDotEnabled` option for letting the user enable or disable the scatter dot. And please note that, there will always be scatter dot displayed on scatter charts. Only line chart, area chart, and step chart can have the option to disable scatter dot.

Reply:
Yes in the screenshot you have above, `AreaChart` should be extending scatter which means the `enabled` flag would be available in there

Reply:
I think I need to figure out what you mean by saying `extending scatter chart`. Both area chart and scatter chart **extend `AxisChart`** and have **scatter component** in them. That's the reason why we can see the scatter dot. And from my understanding, **scatter component** will only be newed in `AxisChart`.
<img width="531" alt="圖片" src="https://user-images.githubusercontent.com/23396970/91964591-e6428e00-ed41-11ea-9aaf-aed4d6d7461c.png">

From what I have observed above, I think whether we want to show or not show the scatter dot, `Scatter` component, should not be configure in `scatterChart` option. `scatterDotEnabled` should be specify in `axisChart` option so all the axis charts with `Scatter` component can access this config option.

Reply:
what @theiliad is saying is that the `ScatterChartOptions` `extends` the `AxisChartOptions` meaning that all charts that have Scatter dots will have `ScatterChartOptions` **in addition to the**  `AxisChartOptions`.

so if you change ScatterChartOptions like this (charts.ts):
```
/**
 * options specific to scatter charts
 */
export interface ScatterChartOptions extends AxisChartOptions {
	/**
	 * options for the points
	 */
	points?: {
		/**
		 * sets the radius of the point
		 */
		radius: number;
		fillOpacity?: number;
		filled?: boolean;
		enabled?: boolean;
	};
}
```

than you can update how you access the config in both scatter-stacked.ts and scatter.ts:
`const isScatterEnabled = Tools.getProperty(this.model.getOptions(), "points", "enabled");`

Reply:
I got what you were saying. However, if I put the `enabled` option in `points` of `ScatterChartOptions` then only scatter chart will be able to access this configuration. How about line chart? We might also want to disable scatter dot in line chart. 
You can see that there is no way for line chart to access `ScatterChartOptions` because of following few lines of code in `charts/line.ts` file. Line chart can only access `LineChartOptions`. Or maybe you actually want to have the `enabled` option being put within `ScatterChartOptions`, `LineChartOptions`, `BubbleChartOptions`, and `AreaChartOptions`? If that is the case, why don't we just have the `enabled` option put within `AxisChartOptions`?
<img width="633" alt="圖片" src="https://user-images.githubusercontent.com/23396970/92866375-11a83580-f432-11ea-8fc5-2218f5f906c0.png">